.npmrc

```diff
@@ -0,0 +1 @@
+shamefully-hoist=true
```

Comment:
It might be worth documenting why we need it, just in case, so we don't forget it's because of shiki patching

Reply:
You're right. Done.